src/modules/flow/py-formatting/py-formatting.json

```diff
@@ -0,0 +1,230 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "py-formatting",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "Apache-2.0",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "converter",
+      "description": "Receives a float packet and converts it to a string one.",
+      "in_ports": [
+        {
+          "data_type": "float",
+          "description": "Where to receive the float value to be converted.",
+          "methods": {
+            "process": "drange_to_string_convert"
+          },
+          "name": "IN"
+        }
+      ],
+      "methods": {
+        "open": "drange_to_string_open",
+        "close": "drange_to_string_close"
+      },
+      "name": "py-formatting/float-to-string-rich-fmt",
+      "aliases": [
+          "converter/float-to-string-rich-fmt"
```

Comment:
"rich-fmt" is strange, the "rich" is not useful, and the "fmt" is an abbreviation that is not easy to co-relate to the formatting module. I'd suggest "converter/float-to-string-format", if you call the module "format". I'd make sure to cross-link both converters to help the user.